arm-graphrbac/1.6-internal/swagger/graphrbac.json

```diff
@@ -1,225 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "title": "GraphRbacManagementClient",
-    "version": "1.6-internal",
```

Comment:
Cleanup other swagger for 1.42-previewInternal as well. We haven't been suing it for a long time now (~ 1 yr or so.)